pyfr/integrators/base.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, backend, rallocs, mesh, initsoln, cfg):
         # Solution gradients cache
         self._curr_grad_soln = None
 
+        # Solution dt cache
+        self._curr_dt_soln = None
```

Comment:
I think it make sense to factor these out into a function, as currently we have them in three places (base, and accept step for both standard and dual controllers).